### ISSUE TITLE
Make accessibility summaries recommended in 1.0

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#2526]
It's been noted that the one issue that prevents a publication that conforms to 1.1 at WCAG 2.0 Level A, AA or AAA from being fully backwards compatible with 1.0 (both the IDPF and ISO versions) is that we reduced the requirement to have a summary to a recommendation. Consequently, only if your 1.1 conforming publication has a summary are you compatible.

This lack of full backwards compatibility with the ISO version may be an issue in some regions.

To fix this situation, all we should need to do is issue an erratum for the 1.0 specification to lower the requirement to a recommendation. It might be worth noting compatibility in the 1.1 specification, although this isn't strictly required.

**Comment by mattgarrish** [comment]
Anything new on being able to modify the IDPF docs, @iherman? If not, would it make sense to make the change first through ISO @murata2makoto @avneeshsingh @GeorgeKerscher?

It's become kind of an awkward gap for the epub/a specification since we have to note this difference between versions while also hinting there's a possible fix for it in the works.

**Comment by avneeshsingh** [comment]
best way would be an errata for IDPF and ISO version.

**Comment by mattgarrish** [comment]
Agree, but the problem is the whole IDPF site has been archived away where we can't modify it, so we haven't been able to access the errata page to make a change: https://idpf.org/epub/a11y/errata/

**Comment by iherman** [comment]
> Anything new on being able to modify the IDPF docs, @iherman? 

To be honest, I do not remember whether there was any discussion on the subject the past year. I cc this to @swickr and @vivienlacourba who, in any case, would "own" the issue, but I believe our starting position is that the IDPF contents are frozen...

**Comment by mattgarrish** [comment]
The alternative to editing the IDPF files would be to get https://idpf.org/epub/a11y/errata/ to redirect somewhere in W3C where we can put a new errata page. But maybe that's no less simple than editing the archived site?

**Comment by iherman** [comment]
> The alternative to editing the IDPF files would be to get https://idpf.org/epub/a11y/errata/ to redirect somewhere in W3C where we can put a new errata page. But maybe that's no less simple than editing the archived site?

Probably the same problem. AFAIK, the IDPF site is essentially read-only...

**Comment by mattgarrish** [comment]
This may be where ISO comes in, then. I don't believe we have any problems getting an erratum done there, so maybe the compatibility section we ultimately add to 1.1 refers to the amended ISO version instead of the original IDPF. It's not like the ISO version is an exact match of the IDPF, so we don't have to issue an erratum on the IDPF before one can be done for the ISO.

We could also note the problem of updating the IDPF version in the 1.1 section and state that the change is applicable to 1.0, even if it can't be a formal erratum. It's not ideal in that it leaves anyone reading the IDPF version unaware of the change, but hopefully most people have migrated to 1.1 by now. We could also broadcast the change through the publishing groups to help spread it out to anyone who hasn't.

**Comment by iherman** [comment]
@mattgarrish raised the issue on the Publ SC meeting yesterday: https://www.w3.org/2024/08/09-pbgsc-minutes.html#t04. I would suggest keeping this open while things settle.

CC @swickr

**Comment by iherman** [comment]
This was discussed during the [pmwg meeting on 18 September 2025](https://w3c.github.io/pm-wg/minutes/2025-09-18.html#69fe).

<details><summary><i>View the transcript</i></summary>

<h3>Issue 2526 - <a href="https://github.com/w3c/epub-specs/issues/2526" rel="noopener noreferrer">w3c/<wbr>epub-specs#2526</a></h3>
<p><cite>wendyreid:</cite> a11y task for has some issues for us<br>
<span>… in 1.1 we dropped summary to recommendation, so we are getting ISO to make summary 1.0 optional</span><br>
<span>… this change is just doing the same thing for the w3c version</span><br>
<span>… I just want to make sure we have agreement before releasing the errata</span></p>
<p><cite>&lt;wendyreid&gt;</cite> Proposed: Update EPUB Accessibility 1.0 to align with EPUB Accessibility 1.1, close issue 2526.</p>
<p><cite>&lt;George&gt;</cite> +1</p>
<p><cite>&lt;mgarrish&gt;</cite> +1</p>
<p><cite>&lt;SueNeu&gt;</cite> +1</p>
<p><cite>&lt;MasakazuKitahara&gt;</cite> +1</p>
<p><cite>&lt;shiestyle&gt;</cite> +1</p>
<p><cite>&lt;gpellegrino&gt;</cite> +1</p>
<p><cite>&lt;duga&gt;</cite> +1</p>
<p><cite>&lt;AvneeshSingh&gt;</cite> +1</p>
<p><cite>&lt;CharlesL&gt;</cite> +1</p>
<p><cite>&lt;wendyreid&gt;</cite> +1</p>
<p><cite>&lt;Hadrien&gt;</cite> +1</p>
<p><cite>wendyreid:</cite> Please vote</p>
<p><cite>&lt;ivan&gt;</cite> +1</p>
<p><cite>&lt;Dale&gt;</cite> +1</p>
<p><strong>RESOLUTION:</strong> Update EPUB Accessibility 1.0 to align with EPUB Accessibility 1.1, close issue 2526.</p>

<hr /></details>
